dispatcher/MongoConnect.py

```diff
@@ -541,10 +541,16 @@ def send_command(self, command, hosts, user, detector, mode="", delay=0, force=F
         Send this command to these hosts. If delay is set then wait that amount of time
         """
         number = None
+        ls = self.latest_status[detector]
         if command == 'stop' and not self.detector_ackd_command(detector, 'stop'):
             self.log.error(f"{detector} hasn't ack'd its last stop, let's not flog a dead horse")
             if not force:
                 return 1
+        for host_list in hosts:
```

Comment:
Yes, it probably should

Reply:
Actually, it doesn't make that much difference. `detector_ackd_command` doesn't use any info from the desired host list here, but it would give us an extra log statement that could help.